client/src/App.tsx

```diff
@@ -20,9 +23,17 @@ const App = () => {
           <Routes>
             <Route path="/" element={<HomePage />} />
             <Route path="/plastic/probleem" element={<PlasticProblemPage />} />
-            <Route path="/plastic/gevolgen" element={<PlasticConsequencesPage />} />
+            <Route
+              path="/plastic/gevolgen"
+              element={<PlasticConsequencesPage />}
+            />
             <Route path="/plastic/aanpak" element={<PlasticApproachPage />} />
             <Route path="/quiz/overview" element={<QuizOverviewPage />} />
+            {/* quiz/login/id is de url die de leerling krijgt om in te loggen */}
```

Comment:
Lijkt mij dat het gewoon een component op /quiz is en dat je daar de code zelf kan invullen.
Daarna krijg je een redirect naar de quiz als je de juiste code hebt.